tracer/src/Datadog.Trace/Logging/Internal/DatadogLoggingFactory.cs

```diff
@@ -30,43 +30,41 @@ internal static class DatadogLoggingFactory
     private const int DefaultRateLimit = 0;
     private const int DefaultMaxLogFileSize = 10 * 1024 * 1024;
 
+    internal const int DefaultConsoleQueueLimit = 1024;
+
     public static DatadogLoggingConfiguration GetConfiguration(IConfigurationSource source, IConfigurationTelemetry telemetry)
     {
         var logSinkOptions = new ConfigurationBuilder(source, telemetry)
                             .WithKeys(ConfigurationKeys.LogSinks)
                             .AsString()
-                           ?.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
+                           ?.Split([','], StringSplitOptions.RemoveEmptyEntries);
 
         FileLoggingConfiguration? fileConfig = null;
         if (logSinkOptions is null || Contains(logSinkOptions, LogSinkOptions.File))
         {
             fileConfig = GetFileLoggingConfiguration(source, telemetry);
         }
 
+        ConsoleLoggingConfiguration? consoleConfig = null;
+        if (logSinkOptions is not null && Contains(logSinkOptions, LogSinkOptions.Console))
+        {
+            consoleConfig = GetConsoleLoggingConfiguration(source);
+        }
+
```

Comment:
I wonder if it wouldn't be better to simplify this:
```suggestion
ConsoleLoggingConfiguration? consoleConfig = logSinkOptions is null || Contains(logSinkOptions, LogSinkOptions.File) ? GetFileLoggingConfiguration(source, telemetry) : null;
ConsoleLoggingConfiguration? consoleConfig = logSinkOptions is not null && Contains(logSinkOptions, LogSinkOptions.Console) ? GetConsoleLoggingConfiguration(source) : null ;

```
Maybe it's not as readable though. 
But it saves lines and the fact that we declare `fileConfig` and `consoleConfig` and set them to `null` just to set them again 2 lines later triggers something in me 😅

Reply:
I totally get this point!
I guess in other contexts I would have been more for separate `if` statements, I just feel like it's kind of dragging the code.

But again, just a personal feeling